index.js

```diff
@@ -71,13 +72,14 @@ app.post('/sets/:setId/fdas', async (req, res) => {
 
 app.get('/querySet', async (req, res) => {
   const { setId, id } = req.query;
+  const service = req.get('Fiware-Service') || req.get('fiwareService');
```

Comment:
```suggestion
  const service = req.get('Fiware-Service');
```

`fiwareService` is not used in our platform